### DISCUSSION TITLE
Urgent simulation for icestudio!

### DISCUSSION THREAD

**Discussion by alico797** [FPGAwars/icestudio#843] [Ideas]
I would like that icestudio had the option of simulation, of the logical states, displays, etc. It would be of great help, especially for those who do not yet have an FPGA board. From my point of view, I think it would be a breakthrough for the program...

**Comment by alico797** [comment]
Me gustaría que icestudio tuviera la opción de simulación, de estados lógicos, displays de 7 segmentos, etc. Sería de gran ayuda, especialmente para aquellos que aún no tienen una placa FPGA. Desde mi punto de vista, creo que sería un gran avance para el programa ...

**Comment by harevalop** [comment]
> Me gustaría que icestudio tuviera la opción de simulación, de estados lógicos, displays de 7 segmentos, etc. Sería de gran ayuda, especialmente para aquellos que aún no tienen una placa FPGA. Desde mi punto de vista, creo que sería un gran avance para el programa ...

Quizá este link pueda ser de ayuda:
https://www.bluetin.io/displays/simulate-oled-lcd-display-pc-pil-opencv/

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
